ocean-client/README.md

```diff
@@ -34,6 +34,8 @@ To run, the script needs parameters set in the AWS ParameterStore:
 /defichain-maxi/settings/min-collateral-ratio
 /defichain-maxi/settings/max-collateral-ratio
 /defichain-maxi/settings/lm-token
+/defichain-maxi/settings/reinvest
+/defichain-maxi/state
```

Comment:
parameter will be created if its not there, so not sure if we need to state it here.

Reply:
for someone who wants to create it himself. it should be listed, i guess.